gef.py

```diff
@@ -7429,6 +7460,8 @@ def __init__(self):
         self.add_setting("clear_screen", True, "Clear the screen before printing the context")
         self.add_setting("layout", "legend regs stack code args source memory threads trace extra", "Change the order/presence of the context sections")
         self.add_setting("redirect", "", "Redirect the context information to another TTY")
+        self.add_setting("libc_args", False, "Show libc function call args description")
+        self.add_setting("libc_args_path", "", "Path to libc function call args json file, provided via gef-extras")
```

Comment:
It actually expects a directory, not a filename

Reply:
file => files